### PR TITLE
Temporary fix for issue https://github.com/syslog-ng/syslog-ng/issues/3178

### PR DESCRIPTION
**Pull request by vivinperis**
module: affile-grammar.ym

Removed the multi-line-timeout option as it was not working

Signed-off-by: Vivin Peris vivinperis98@gmail.com

### CODE REVIEW

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by alltilla** [comment]
@kira-syslogng ok to test

**Comment by kira-syslogng** [comment]
Build FAILURE

File: modules/affile/affile-grammar.ym [hunk 1]
```diff
@@ -260,10 +260,6 @@ multi_line_option
 	    CHECK_ERROR_GERROR(log_proto_multi_line_server_options_set_garbage(&last_log_proto_options->super, $3, &error), @3, error, "error compiling multi-line regexp");
 	    free($3);
 	  }
-	| KW_MULTI_LINE_TIMEOUT '(' nonnegative_integer ')'
```
Comment (alltilla) [inline modules/affile/affile-grammar.ym]: This is a good approach, but not quite the ideal one, it has one problem:
Removing `KW_MULTI_LINE_TIMEOUT` from `multi_line_option` removes it from other users of `multi_line_option`, like `source_affile_option`.

Please only remove it from `source_afpipe_option`.

**Comment by alltilla** [comment]
travis-ci fails because of the commit message:
> Please enter the commit message for your changes. Lines starting

Please fix it according to https://github.com/syslog-ng/syslog-ng/blob/master/CONTRIBUTING.md#commits

Thanks! :)

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by vivinperis** [comment]
Whats the issue now?

**Pull request closed without merging**